server/src/main/java/org/room/apollo/server/dto/deezer/ChartD.java

```diff
@@ -0,0 +1,33 @@
+package org.room.apollo.server.dto.deezer;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.room.apollo.server.converter.PlaylistDeserializer;
+import org.room.apollo.server.converter.TracksDeserializer;
+
+import java.util.List;
+
+public class ChartD {
+    @JsonDeserialize(using = TracksDeserializer.class)
+    private List<TrackD> tracks;
+    @JsonDeserialize(using = PlaylistDeserializer.class)
+    private List<PlaylistD> playlists;
+//    private List<ArtistD> playlists;
+//    private List<AlbumD> playlists;
```

Comment:
maybe remove this commented code, and from other classes as well.

Reply:
i will use it in future

Reply:
ok...